app/assets/src/components/PipelineSampleReport.jsx

```diff
@@ -15,7 +15,10 @@ import OurDropdown from "./ui/controls/dropdowns/Dropdown";
 import MultipleDropdown from "./ui/controls/dropdowns/MultipleDropdown";
 import ThresholdFilterDropdown from "./ui/controls/dropdowns/ThresholdFilterDropdown";
 import BetaLabel from "./ui/labels/BetaLabel";
+import PathogenLabel from "./ui/labels/PathogenLabel";
+import PathogenSummary from "./views/report/PathogenSummary";
 import PhyloTreeInputs from "./views/phylo_tree/PhyloTreeInputs.jsx";
+import ReportInsightIcon from "./views/report/ReportInsightIcon";
```

Comment:
Can you add this to `ui/icons` and call it BulbIcon or something more descriptive of the image?